### PR TITLE
VS-1000 - Migrate VQSR Classic Code to own WDL

### PR DESCRIPTION
**Pull request by gbggrant**
Migrate VQSR Classic code to its own WDL as much as possible (to make it easier to rip out some sunny day)

### CODE REVIEW

**Comment by gbggrant** [comment]
Passing Integration Test [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/70870e97-4e58-4a55-8424-f78b0e26ac28).

File: scripts/variantstore/wdl/GvsUtils.wdl [hunk 1]
```diff
@@ -864,3 +864,86 @@ task SummarizeTaskMonitorLogs {
     File monitoring_summary = "monitoring_summary.txt"
   }
 }
+
+# Note - this task should probably live in GvsCreateFilterSet, but I moved it here when I was refactoring VQSR Classic out of
+# GvsCreateFilterSet (in order to avoid a circular dependency)
+# When VQSR Classic is removed, consider putting this task back in GvsCreateFilterSet
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsUtils.wdl:870]: Maybe worth adding as a comment to https://broadworkbench.atlassian.net/browse/VS-992?

**Review by rsasch — COMMENTED** [review]
Would you consider taking the tasks from GvsWarpTasks.wdl and adding them to GvsVQSRClassic.wdl so that we can have one fewer WDL (and also just one file to delete when VQSR classic is retired)?

**Comment by rsasch** [comment]
also, #1000 🎉 !

**Comment by gbggrant** [comment]
> Would you consider taking the tasks from GvsWarpTasks.wdl and adding them to GvsVQSRClassic.wdl so that we can have one fewer WDL (and also just one file to delete when VQSR classic is retired)?

This is a good idea.

File: scripts/variantstore/wdl/GvsUtils.wdl (hunk 1 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsUtils.wdl:870]: So commented.

**Review by rsasch — APPROVED** [review]
LGTM, assuming latest integration run succeeds 👍🏻

**Comment by gbggrant** [comment]
Integration test [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/80b6ed98-3b79-40f3-b55b-524b6f63f1e1).

Failed one of the four (in Hail VDS tieout) - with a weird python3 error. 99.9994% sure not related to this PR.

**Pull request merged**